crates/compiler/gen_dev/src/generic64/mod.rs

```diff
@@ -2780,6 +2825,72 @@ impl<
 
                 self.free_symbol(&tmp);
             }
+            UnionLayout::NullableWrapped {
+                nullable_id,
+                other_tags,
+            } => {
+                let number_of_tags = other_tags.len() + 1;
+
+                let dst_reg = self.storage_manager.claim_general_reg(&mut self.buf, sym);
+
+                // build a table to index into with the value that we find
+                let nullable_id = *nullable_id as usize;
+                let it = std::iter::once(nullable_id)
+                    .chain(0..nullable_id)
+                    .chain(nullable_id + 1..number_of_tags);
```

Comment:
Just to make sure I understand. lets say we had the tags 0 to 5 and 3 was the nullable id. This would be 3, 0, 1, 2, 4, 5. Then you just index into it. to get the correct number?

Of course long term we would probably store this as a static table instead of building it on the fly. That way, we don't have to pay a cost based on the number of tags each time we hit this.

Reply:
yes, exactly. And yes this would ideally be a global table, but I don't know how to do that, and getting this to work seemed more important